deploy/examples/create-external-cluster-resources.py

```diff
@@ -18,18 +18,13 @@
 import sys
 import json
 import argparse
-import unittest
 import re
 import requests
 import subprocess
 from os import linesep as LINESEP
 from os import path
 
-# backward compatibility with 2.x
-try:
-    ModuleNotFoundError
-except:
-    ModuleNotFoundError = ImportError
+ModuleNotFoundError = ImportError
```

Comment:
If we are no longer supporting python 2, we may need to put a shebang at the top of the file specifying python 3. It's possible this is the cause of the CI failures.

`#!/usr/bin/env python3`

Reply:
CI was failing because we don't have the `import unitest` header, it will fix now,
In master we had move the unit test that's why it worked there